src/components/navigation/sidebar.tsx

```diff
@@ -12,8 +12,8 @@ export default function Sidebar() {
       <div className="space-y-4 py-4">
         <div className="px-3 py-2">
           <div className="space-y-1">
-            <h2 className="mb-2 px-4 text-xl font-semibold tracking-tight">
-              Overview
+            <h2 className="mb-2 px-4 text-2xl font-semibold tracking-tight pb-4">
+              Armitage
```

Comment:
This is correct?

Reply:
yeah, this is the title of the menu bar, made no sense before that it was "Overview" on the menu, it should be our name 😅 
![image](https://github.com/armitage-labs/armitage-monorepo/assets/6979719/e2d4efcf-b8ac-4739-b4b6-b452745dbd3a)